docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md

```diff
@@ -129,16 +131,11 @@ spec:
       enable: true
 ```
 
-### Step 2: Enable `public-api` Plugin
+## Expose metrics with the public-api Plugin
 
-Let's make a basic routing setup, and please note that further configuration should be done based on your local backend service information. The primary solution concept is to use the `public-api` plugin to protect the routes exposed by *Prometheus*. For a more detailed configuration, you can refer to the [example](https://apisix.apache.org/docs/apisix/plugins/public-api/#example) section of the `public-api` plugin.
-
-```bash
-kubectl apply -f prometheus-route.yaml -n ingress-apisix
```

Comment:
If you want to delete this line, please add the description of the specified namespace. Or specify the namespace in yaml.

Reply:
Have you tried this tutorial?

Reply:
> Have you tried this tutorial?

Yes. I always try the tutorial/docs pages to make sure that there are no errors or if we need to make any updates.

> If you want to delete this line, please add the description of the specified namespace. Or specify the namespace in yaml.

Yes. Makes sense. Explicitly mentioning the namespace is a good idea. I will add it.

Reply:
@AlinsRan Is it okay now?